packages/tocco-ui/src/FormattedValue/typeFormatters/TimeFormatter.js

```diff
@@ -27,7 +27,12 @@ const TimeFormatter = props => {
 
 TimeFormatter.propTypes = {
   intl: intlShape.isRequired,
-  value: PropTypes.object.isRequired
+  value: PropTypes.shape({
+    hourOfDay: PropTypes.number,
+    minuteOfHour: PropTypes.number,
+    secondOfMinute: PropTypes.number,
+    millisOfSecond: PropTypes.number
```

Comment:
The naming of the properties is not consistent. why are hours, minutes and seconds named in singular and millis in plural?